docs/dictionary-data.md

```diff
@@ -201,9 +122,13 @@ The fields are:
   - The `senses` field, a required array of `{definition: string, sources:
     string[]}` objects, contains the definitions for the entry.
 
-    The sense must currently be a single unformatted string. We are aware
-    that people would like to specify things such as source-language text
-    to be shown in the current orthography, cross-references, notes,
+    If we were starting from scratch we might call the `definition` field a
+    `displayDefinition`, but we already have some data, and often it is the
+    only one definition field provided.
+
+    The definition must currently be a single unformatted string. We are
+    aware that people would like to specify things such as source-language
+    text to be shown in the current orthography, cross-references, notes,
     examples, and so on.
```

Comment:
This suggestion more reflects what I'd _hope_ would be the case. I think it'd get really bad if we kept trying to cram more things into the definition fields.

```suggestion
    The definition must currently be a single unformatted string. We are
    aware that people would like to specify things such as source-language
    text to be shown in the current orthography, cross-references, notes,
    examples, and so on. If these are supported in the future, they will
    likely be separate fields rather included within the definition.
```